docs/compatibility.md

```diff
@@ -0,0 +1,24 @@
+# Compatability
+
+## Linux
+
+There aren't any known compatability issues on Linux.
+
+## OS X
+
+There aren't any known compatability issues on OS X.
+
+## Windows
+
+PyGradle has basic support for Windows. We would like to note that pex is not
+offically supported on Windows, while it seems to work properly we cannot
+guarantee it.
+
+We're eager to review pull requests from our Windows user base!
+
+### Caveats
+
+- The integration tests require that Python be available in your PATH.
+- The pex format defaults to fat.
```

Comment:
maybe add a blurb here about fat pex vs thin pex, as this is our own nomenclature (not terminology used by other pex users)